connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -135,7 +134,7 @@ public void stop() {
             WorkerConnector conn = entry.getValue();
             log.warn("Shutting down connector {} uncleanly; herder should have shut down connectors before the" +
                     "Worker is stopped.", conn);
-            conn.stop();
+            conn.shutdown();
```

Comment:
Is the terminology changing for a reason? We've used `stop` pretty much everywhere else.